Makefile

```diff
@@ -14,3 +14,52 @@ compose_up:
 
 compose_down:
 	docker-compose -f docker/docker-compose.yml down
+
+# Development Setup
+.venv:
+	python3 -m venv .venv
+	. .venv/bin/activate
+	python3 -m pip install -r requirements-dev.txt
+
+.git/hooks/pre-commit: .venv
+	python3 -m pre_commit install
+
+.env:
+	# afterwords, set in .env
+	# TAKAHE_DATABASE_SERVER="postgres://postgres:insecure_password@localhost:5433/takahe"
+	cp development.env .env
+
+_PHONY: python-dev
+python-dev: .venv
+
+_PHONY: precommit
+precommit: python-dev .git/hooks/pre-commit
+
+_PHONY: startdb stopdb
+startdb:
+	docker compose -f docker/docker-compose.yml up db -d
+
+stopdb:
+	docker compose -f docker/docker-compose.yml stop db
+
+_PHONY: superuser
+superuser: .env python-dev startdb
+	python3 -m manage createsuperuser
+
+_PHONY: pydev
+pydev: python-dev precommit
+
+_PHONY: test
+test: pydev
+	python3 -m pytest
+
+# Active development
+_PHONY: migrations server stator
+migrations: startdb
+	python3 -m manage migrate
+
+server: startdb
```

Comment:
Can these be `runserver` and `runstator` instead? I like makefile actions to sound more like what they're doing.

Reply:
sure thing